features/utilities/src/main/resources/features.xml

```diff
@@ -77,19 +77,6 @@
         <bundle>mvn:ddf.mime.core/mime-core-configurableresolver/${project.version}</bundle>
         <bundle>mvn:org.apache.tika/tika-core/${tika.version}</bundle>
         <bundle>mvn:org.codice.thirdparty/tika-bundle/${tika.thirdparty.bundle.version}</bundle>
-
-        <configfile finalname="${ddf.etc}/DDF_Custom_Mime_Type_Resolver-csw.config">
```

Comment:
Do the default configurations without a config file work for these features. I dislike the idea of having a feature that needs some configuration which isn't installed with the feature just to work around an issue with Itests. That being said you could install it here and have it already exist in the itests.

Reply:
Yeah I don't like this either. I'm thinking about creating a new module (or modules) that these configs  live under so that they can still be referenced in the features files. I'll be looking at that again after I get the features tests passing in CI

Reply:
Something to consider is this should make it easier for downstream distributions to remove or override managed service factory configurations.

Reply:
There is that 😄 
As it stands with these changes all the defaults will be inherited as long as none of the configs are getting excluded from a custom distribution when pulling in ddf-common

Reply:
We could also deploy those config files from the ddf-common and keep the features as is.